### DISCUSSION TITLE
x-bind:asp-action not working

### DISCUSSION THREAD

**Discussion by melakhvision** [alpinejs/alpine#3870] [1. Help]
Hello 
I have added alpine in my .Net 7 MVC project everything is working except x-bind:asp-action.
If I use :bind:asp-action it doesn't work either.
On the dev tools console I the the value of asp-action being populated two times but it doesn't work

```html
<template x-for="item in menuItems" :key="item.id">
    <li :key="item.id">
        <a :asp-action="item.action" x-bind:asp-controller="item.controller" class='text-uppercase'
            x-text="item.name"></a>
    </li>
</template>
```
On the dev console the value is shown two time. One with the expected value another with the code but navigating is not working.
![SHOW TO TIMES](https://github.com/alpinejs/alpine/assets/45154979/b1020584-23b2-4cf9-abf5-6ee70a625c28)

**Comment by ekwoka** [comment]
Duplicate #3871
